### PR TITLE
How to Detect and Correct Multicollinearity in Regression Models

### PR DESCRIPTION
**Pull request by collins-a**
Article as index.md and hero image as hero.jpg

### Article Checklist
**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process, quality, and style-once published, you can then submit any number of articles.)**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [ ] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by collins-a** [comment]
This fixes #1330

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by hectorkambow** [comment]
can we get a status on this PR ? @lalith1403

**Comment by lalith1403** [comment]
The technical portions covered in this article are very good. Will update by EOD.

File: articles/multicollinearity/index.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+**More data.** Statistically, a regression model with more data is likely to suffer less variance due to a larger sample size. This reduces the impact of multicollinearity.
+
+**Partial least squares regression (PLS) or principal component analysis (PCA).** A takeaway from this paper on [partial least squares regression](http://www.ijastnet.com/journals/Vol_4_No_1_January_2014/17.pdf) for multicollinearity is that PLS can lessen variables to a smaller grouping with no correlation between them. PLS, like PCA, is a dimensionality reduction technique. PCA reduces the dimension of data through the decomposition of data into independent factors. Resultantly, new variables with no correlation between them are created. This [article](https://towardsdatascience.com/how-to-remove-multicollinearity-in-dataset-using-pca-4b4561c28d0b) explains how PCA handles multicollinearity.
```
Comment (lalith1403) [inline articles/multicollinearity/index.md:100]: The heading needs to changed... Are you suggesting that PLS and PCA are the same? It needs to be made clear that PLS is similar to PCA... The heading is misleading.

File: articles/multicollinearity/index.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+1. The inclusion of identical variables. For example, one may have identical variables in a dataset, such as mass in kilograms and mass in pounds.
+
+2. Creation of new variables that are dependent on others. When we create variables that depend on other variables, we introduce redundant information to the model. As such, we may inadvertently encourage the occurrence of multicollinearity.
+
+3. Inadequate data. In some cases, when the data is inadequate, we may experience multicollinearity. This is due to the small sample size, which might, in turn, experience great variance.
+
+#### Variance Inflation Factor
+
+To learn the severity of multicollinearity, there are a few tests that may be carried out. We focus on the use of the variance inflation factor (VIF). The variance inflation factor represents the quotient of model variance with many terms against model variance with a single term. It is the score of an independent variable representing the degree to which other independent variables explain the variable. As such, it works out the magnitude of the correlation between independent variables.
```
Comment (lalith1403) [inline articles/multicollinearity/index.md:67]: Can you explain the definition a bit more detailed manner. The definition is accurate for VIF, but I feel it needs more explanation in simpler words or through the help of an example.

File: articles/multicollinearity/index.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+**Partial least squares regression (PLS) or principal component analysis (PCA).** A takeaway from this paper on [partial least squares regression](http://www.ijastnet.com/journals/Vol_4_No_1_January_2014/17.pdf) for multicollinearity is that PLS can lessen variables to a smaller grouping with no correlation between them. PLS, like PCA, is a dimensionality reduction technique. PCA reduces the dimension of data through the decomposition of data into independent factors. Resultantly, new variables with no correlation between them are created. This [article](https://towardsdatascience.com/how-to-remove-multicollinearity-in-dataset-using-pca-4b4561c28d0b) explains how PCA handles multicollinearity.
+
+**Centering the variables.** Centering is defined as subtracting a constant from the value of every variable. It redefines the zero point for a given predictor to become the value we subtracted. Here’s how we can center the variables. We can first calculate the mean of every single independent variable. The next step would replace each variable value with the difference between the values and the mean. This reduces the impact of multicollinearity.
```
Comment (lalith1403) [inline articles/multicollinearity/index.md:102]: How does this impact multicollinearity?

**Review by lalith1403 — CHANGES_REQUESTED** [review]
A top notch article on multicollinearity. A few additions and this should be a very good article ready to be published.

**Comment by collins-a** [comment]
@lalith1403 thanks, sir. Added a few lines to address the changes. Let me know if they are sufficient. Cheers.

**Review by lalith1403 — APPROVED** [review]

**Comment by hectorkambow** [comment]
👋  Fantastic article @collins-a  - really learned a lot about Multicollinearity - think the community will find it useful. 
I'll be sure to revisit it again. 

Just wanted to give you an FYI that there were a few parts that I found a bit difficult to great through - the intro and a few other sentences. (as such I will be placing a tiered label for payouts)

I changed what I could to help the article flow. Please go through my changes to make sure I omitted nothing important.  
Also to see apply any applicable changes moving forward to future articles (or revisions). 

cc @lalith1403  please do the same to improve future revisions 👍  

Overall fantastic job! 
Thank you gentlemen 🚀

**Pull request merged**